generators/app/templates/src/containers/TicTacToe/GameListPage/children/Form/game-form.component.js

```diff
@@ -0,0 +1,237 @@
+      ]);
+      /**
+       * If opponent has at least one inbox will create a game and send notification
+       * if not will show an error.
+       */
+      if (inboxes.length > 0) {
+        const newDocument = await ldflexHelper.createNonExistentDocument(documentUri);
+        if (newDocument) {
+          const document = await ldflexHelper.fetchLdflexDocument(documentUri);
+          const setupObj = initialGame(opponent);
+
+          for await (const field of tictactoeShape.shape) {
+            const prefix = tictactoeShape['@context'][field.prefix];
+            const predicate = `${prefix}${field.predicate}`;
+            await document[predicate].add(setupObj[field.predicate]);
+          }
+          /**
+           * Find opponent game inbox if doesn't exist get global
+           * @to: Opponent inbox path
+           */
+          const to = helperNotification.getDefaultInbox(inboxes, 'Game', 'Global');
+          const target = `${window.location.href}/${btoa(documentUri)}`;
+          await sendNotification(
+            {
+              title: 'Tictactoe invitation',
+              summary: 'has invited you to play Tic-Tac-Toe.',
+              actor: webId,
+              object: documentUri,
+              target
+            },
+            to.path
+          );
+
+          setDocumentUri(`${Date.now()}.ttl`);
+        } else {
+          errorToaster(`${opponent} ${t('game.createFolder')}`, 'Error');
+        }
+      } else {
+        errorToaster(`${opponent} ${t('noInboxOpponent.message')}`, 'Error', {
+          label: t('noInboxOpponent.link.label'),
+          href: t('noInboxOpponent.link.href')
+        });
+      }
+    } catch (e) {
+      throw new Error(e);
+    }
+  };
+
+  const createACLFile = async (documentUri, body) => {
+    try {
+      ldflexHelper.createDocumentWithTurtle(documentUri, body);
+    } catch (e) {
+      throw new Error('Error while creating ACL');
+    }
+  };
+
+  const createQuad = (subject, predicate, object) => ({
+    subject,
+    predicate,
+    object
+  });
+
+  const createDefaultQuads = (modes, webId, subject, prefixes) => {
+    const { acl, foaf, a } = prefixes;
+    const originalPredicates = [
+      createQuad(subject, `${a}`, namedNode(`${acl}Authorization`)),
+      createQuad(subject, `${acl}accessTo`, namedNode(documentUri))
+    ];
+    const predicates = webId
+      ? [...originalPredicates, createQuad(subject, `${acl}agent`, namedNode(webId))]
+      : [...originalPredicates, createQuad(subject, `${acl}agentClass`, namedNode(`${foaf}Agent`))];
+
+    const quadArray = modes.reduce(
+      (array, mode) => [...array, createQuad(subject, `${acl}mode`, namedNode(`${acl}${mode}`))],
+      predicates
+    );
+
+    return quadArray;
+  };
+
+  const aclTurtle = async (documentUri, opponent) => {
+    try {
+      const aclDocument = `${documentUri}.acl`;
+      const { DataFactory } = N3;
+      const { namedNode, quad } = DataFactory;
+      const prefixes = {
```

Comment:
We should be storing these in a re-usable file somewhere instead of re-making the prefixes

Reply:
This will be handled differently with the new ACL library